tests/cpp/test_matmul.cpp

```diff
@@ -3663,7 +3663,7 @@ TEST_F(HopperMatmulTest, HSH_NT_128BSwizzle) {
   const int64_t cta_m = 2 * getM(macro);
   const int64_t cta_n = 1 * getN(macro);
 
-  constexpr std::tuple<int64_t, int64_t, int64_t> cluster_dims{2, 1, 1};
+  constexpr std::tuple<int, int, int> cluster_dims{2, 1, 1};
```

Comment:
super nitpick: can we stick with `int64_t` for consistency?
```suggestion
  constexpr std::tuple<int64_t, int64_t, int64_t> cluster_dims{2, 1, 1};
```

Reply:
Yeah, I was mostly doing that because the `MatmulParams` entries are `int`, but we should probably just change `MatmulParams` instead (in another PR).

Reply:
Done. Much smaller PR now...